### PR TITLE
Launch preps: Location

### PR DESCRIPTION
**Pull request by dns2utf8**
I started rewriting as much as I could figure out with 30min of searching the internet.
What needs to be done:

- [x] Some CC or CC0 banner
- [ ] Introduction to the city
- [ ] Travel examples for popular destinations by train (eg. Zürich, Rome, Madrid, ...)
- [x] Entry/Visa requirements
- [x] Travel directions inside Paris
- [ ] Add a picture of the venue
- [ ] Update addition information

### CODE REVIEW

**Comment by Hoverbear** [comment]
I'm worried the picture is quite small. Maybe we can use one from https://unsplash.com/search/photos/paris ? They are free to use.

File: location.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ With three state universities, many tech companies ranging from small startups t
 </section>
 
 <section class="img-grid">
-  <img src="/assets/zurich/zurich-panorama.jpg" alt="Zurich purple hour (panorama shot)">
+  <img src="/assets/paris/paris-panorama.jpg" alt="Paris Frankreich Eiffelturm Nacht (panorama shot)">
```
Comment (Hoverbear) [inline location.md]: This is in German ^^

File: location.md [hunk 2]
```diff
@@ -42,45 +42,35 @@ People looking for cheap flights might also consider flying to the *EuroAirport
...
 ## Additional information
 
 <ul>
   <li>Yes, you can drink the tap water.</li>
-  <li>The local currency is CHF (not EUR). Zürich is generally a card friendly city.</li>
-  <li>The local power socket is not the EU one. You may need an adaptor.</li>
-  <li>Zürich is generally fairly expensive compared to other European cities, be prepared to pay a bit more for your favorite döner.</li>
+  <li>The local currency is € (EUR).</li>
+  <li>The local power socket is the EU one. You may need an adaptor.</li>
+  <li>Paris is generally fairly expensive compared to other European cities, be prepared to pay a bit more for your favorite döner.</li>
```
Comment (Hoverbear) [inline location.md]: döner -> Baguette?

**Comment by Hoverbear** [comment]
@ldoguin could you maybe help us write something nice about Paris here?

**Comment by GuillaumeGomez** [comment]
I can add some information about how to get to the event place if you want.

**Comment by dns2utf8** [comment]
That would be great 👍

**Comment by GuillaumeGomez** [comment]
I'll do it in the next days or so (don't hesitate to ping if I forget!).

**Comment by skade** [comment]
If the venue info isn't fully fleshed out, it shouldn't hold us back. We can patch that ;).

**Comment by GuillaumeGomez** [comment]
I added some information (and checked two checkboxes).

**Comment by Hoverbear** [comment]
Merging in progress data in prep for launch. We still need to do more though =D

**Pull request merged**